src/ripple/app/ledger/Ledger.test.cpp

```diff
@@ -223,7 +223,7 @@ class Ledger_test : public beast::unit_test::suite
     Ledger::pointer
     close_and_advance(Ledger::pointer ledger, Ledger::pointer LCL)
     {
-        SHAMap::pointer set = ledger->peekTransactionMap();
+        std::shared_ptr<SHAMap> set = ledger->peekTransactionMap();
```

Comment:
peekTransactionMap returns a ref. You could take advantage and make set a const ref.